### PR TITLE
c2f10 crop and soil inputs

### PR DESCRIPTION
**Pull request by Rebecavb**
Cohort 2 farm 10 inputs

### CODE REVIEW

**Review by jadamchick — APPROVED** [review]
Hi Rebeca! Looks good, the only thing is can you please not include the updates to default_task.json?

File: input/data/tasks/default_task.json
Comment (jadamchick) [inline input/data/tasks/default_task.json:1]: can you please revert this back to the default before merging? thanks!

**Review by krb78 — APPROVED** [review]
Looks great! The yields aren't what they should be even with the irrigation - but this might get fixed (I know hydrology of the soil layers is still being worked out).

**Comment by Rebecavb** [comment]
> Hi Rebeca! Looks good, the only thing is can you please not include the updates to default_task.json?

Hi Julie, yes absolutely. Thanks for the review.

**Pull request merged**